### PR TITLE
Even more bug food. You will eat the bugs II: Electric Boogaloo.

### PR DESCRIPTION
**Pull request by Eneocho**
## The Gunkening is coming:
Adds even more cockroach/cricket based dishes. 
Cockroach salad, made with 2 cabbages, one lime and 2 slabs of cockroach meat.
roachsalad
Gunk Soup, made with 10u Water, 5u Gunky Chemical and 2 slabs of cockroach/cricket meat.
gunksoup
Gunk Bread, made with 15u Flour, 5u Gunky Chemical, 3 slabs of mutated cockroach meat and 3 cheese wedges/moon cheese.
gunkbread
Gunk Pie, made with 5u Flour, 5u Gunky Chemical and 1 slab of cockroach/cricket meat.
gunkpie
Gunk Cake, made with 15u Flour, 5u Gunky Chemical, 2 slabs of mutated cockroach meat and 3 eggs.
gunkcake
Roaches-on-a-stick, made with 1 rod, 2 slabs of cockroach meat, 5u Gunky Chemical and 1 egg.
roachesonstick
Grandpa Tik's Roasted 'Peanuts', available under contraband in your nearest Getmore vendor.
grandpatik

:cl:
 * rscadd: Gunky chemicals include: Chemical Waste, Toxic Waste, Vomit, Toxin, Solanine, Radium, Mutagen, Untable Mutagen and Spiders.
 * rscadd: Cockroach Salad, made with 2 cabagges, one lime and 2 cockroach meat.
 * rscadd: Gunk Soup, made with 10u Water, 5u Gunky Chemical and 2 cockroach/cricket meat.
 * rscadd: Gunk Bread, made with 15u Flour, 5u Gunky Chemical, 3 cheese wedges and 3 mutated cockroach meat.
 * rscadd: Gunk Pie, made with 5u flour, 5u Gunky Chemical and 1 slab of cockroach/cricket meat.
 * rscadd: Gunk Cake, made with 15u Flour, 5u Gunky Chemical, 3 eggs and 2 slabs of mutated cockroach meat.
 * rscadd: Roaches on a stick, made with 1 rod, 1 egg, 5u Gunky Chemical and 2 cockroach meat.
 * rscadd: Grandpa Tik's Roasted 'Peanuts', a special blend of cockroach, cricket and peanut, available under contraband in your nearest Getmore vendor.
 * tweak: All gunkburger types and gunk kabob have some amount of cockroach chitin inside.
 * tweak: Super Gunk Burgers now need 10u Gunky Chemical instead of 5u Salt and 5u Pepper.

### CODE REVIEW

**Comment by WindowsErrors** [comment]
It's 2030

Your house is 99 degrees and you can't put on the air conditioning because the governor declared an emergency

You get into your EV to go to the bar but the screen says it wasn't charging, governor's orders

You decide to walk to a local park but a cop stops you, saying it's illegal to leave your quarantine zone

You head back and sit outside your dwelling, then grab a bag of dried crickets to snack on and wait for the temperature to drop

**Comment by Kurfursten** [comment]
Mixed feelings on these recipes. Gunk food feels like it should be low-tech but radium requires you to go get a chem dispenser, and chemical waste requires either grinding dan food or access to a TEG and space lube (chem dispenser). Getmore also doesn't seem like it should sell gunk food, seems like a dan food.

**Comment by Eneocho** [comment]
Radium is really simple to get, just plant some glowshrooms and grind, no chem dispenser required. I can change it if you want, but I wanna keep the chemwaste on these ones, else they're gunk in name only, since bugs by themselves aren't really "gunky".
Also, a slightly unknown method for manufacturing chem waste is to use a laser scalpel on your lube. No TEG required, it just needs to be hot, or I guess mixing sodium poly with lube or water (works in the beaker).
Either way, they're not exceedingly difficult chems to get, you can ask the botanist for glowshrooms and get your radium and ask the chemist for sodium poly or lube. We already have a few recipes which call for lube or radium.
The Nuts n Bugs one isn't intended to be gunk food, it's highly nutritious processed bugs mixed with nuts, no toxins, no garbage, no waste. Just nutritious bugs and peanuts. It's under contraband since not many would admit to eating bugs (which is kinda the point).

**Comment by tinysaturn** [comment]
Kinda wish the pie, soup and bread looked more unique

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 1]
```diff
@@ -7976,3 +7879,231 @@ var/global/list/bomb_like_items = list(/obj/item/device/transfer_valve, /obj/ite
...
+
+/obj/item/weapon/reagent_containers/food/snacks/gunksoup
+	name = "Gunk Soup"
+	desc = "Smells like a garbage can."
+	icon_state = "gunksoup"
+	trash = /obj/item/trash/snack_bowl
+	food_flags = FOOD_MEAT | FOOD_LIQUID
+	crumb_icon = "dribbles"
+	filling_color = "#6D4930"
+	valid_utensils = UTENSILE_FORK|UTENSILE_SPOON
```
Comment (pedr69) [inline code/modules/reagents/reagent_containers/food/snacks.dm:8011]: >gunk soup
>it's so thick you can eat it with a fork

jesus christ how horrible

**Comment by Cartlord** [comment]
why is this good for the game

**Comment by Hinaichigo** [comment]
I think maybe these and the other gunk recipes and stuff could use less reliance on traditional ingredients like eggs and flour, instead putting more gross stuff in them like bug eggs and cricket flour. However I'm not versed in bug farming so I don't know what's available.

Also if it's possible with the recipe system, maybe to add allow a variety of reagents instead of eg. solely chemical waste. Like using the broad "toxins" category or maybe even a new one like "gunk/waste liquids" or something. 

Just really unhealthy gross cakes made of bugs and goo.

**Comment by Eneocho** [comment]
> why is this good for the game

Gives more uses to otherwise useless meat types, and roleplay (like most foods).

> I think maybe these and the other gunk recipes and stuff could use less reliance on traditional ingredients like eggs and flour, instead putting more gross stuff in them like bug eggs and cricket flour. However I'm not versed in bug farming so I don't know what's available.

You're onto something here. Cockroach eggs would fit perfectly. There's no such thing as cricket flour here, crickets just give regular flour when processed.

> Also if it's possible with the recipe system, maybe to add allow a variety of reagents instead of eg. solely chemical waste. Like using the broad "toxins" category or maybe even a new one like "gunk/waste liquids" or something.

Not sure if the recipe system can handle categories instead of just independent ingredients. I'll check it out. If it works I'll implement it that way.

**Comment by Hinaichigo** [comment]
> VOMIT

Nice.

**Comment by Hinaichigo** [comment]
What's the reasoning why roaches on a stick requires an egg. Is it basted onto the roaches?

**Comment by Eneocho** [comment]
> What's the reasoning why roaches on a stick requires an egg. Is it basted onto the roaches?

Yes. Though if you want I can take it out.

**Comment by Hinaichigo** [comment]
It's all good just wondering.

**Comment by BomberBro** [comment]
Nuts and Bugs should have something other than a live cockroach on the cover if it's going under the getmore brand. Or just call it protein bites with the green bag and have the bug part be implied.

**Comment by Eneocho** [comment]
> Nuts and Bugs should have something other than a live cockroach on the cover if it's going under the getmore brand. Or just call it protein bites with the green bag and have the bug part be implied.

I'll try to make a better sprite.

**Comment by Eneocho** [comment]
> Nuts and Bugs should have something other than a live cockroach on the cover if it's going under the getmore brand. Or just call it protein bites with the green bag and have the bug part be implied.

bug
Is this better?

**Pull request merged**